### DISCUSSION TITLE
%EB%B2%88%EC%95%84%EC%9B%83%EC%9D%84-%ED%95%9C-%EB%B2%88%EC%97%90-%ED%95%B4%EA%B2%B0%ED%95%A0-%EC%88%98-%EC%9E%88%EB%8A%94-%EB%B0%A9%EB%B2%95%EC%9D%80-%EC%84%B1%EA%B3%BC%EB%8B%A4/

### DISCUSSION THREAD

**Discussion by giscus** [heli-os/profile-giscus#10] [Comments]
# %EB%B2%88%EC%95%84%EC%9B%83%EC%9D%84-%ED%95%9C-%EB%B2%88%EC%97%90-%ED%95%B4%EA%B2%B0%ED%95%A0-%EC%88%98-%EC%9E%88%EB%8A%94-%EB%B0%A9%EB%B2%95%EC%9D%80-%EC%84%B1%EA%B3%BC%EB%8B%A4/

볼타 CTO 테오 블로그

https://dataportal.kr/%EB%B2%88%EC%95%84%EC%9B%83%EC%9D%84-%ED%95%9C-%EB%B2%88%EC%97%90-%ED%95%B4%EA%B2%B0%ED%95%A0-%EC%88%98-%EC%9E%88%EB%8A%94-%EB%B0%A9%EB%B2%95%EC%9D%80-%EC%84%B1%EA%B3%BC%EB%8B%A4/

**Comment by DFDRDODM95** [comment]
포스트 개발 조직에서의 리더 중 
'변화를 저지하는 것도 리더십' 이라는 문구와 
통하는 부분이 있는 거 같습니다. 👍
